src/elfpy/utils/sim_utils.py

```diff
@@ -24,6 +26,61 @@
     from elfpy.agent import Agent
 
 
+def get_simulator(config: Config, rng: Generator, agents: list[Agent] = []) -> Simulator:
+    """Constructs a simulator with sane defaults and initializes the simulator
+    with an initial LP.
+
+    Arguments
+    ---------
+    rng : Generator
+        The random number generator that should be used in these simulations.
+        We pass this in to avoid restarting the generator after using it to
+        setting up the agents.
+    config : Config
+        The simulator config.
+    agents : list[Agent]
+        The agents to that should be used in the simulator.
+    """
+    # Sample the random simulation arguments.
+    random_sim_vars = get_random_variables(config, rng)
```

Comment:
It looks like you moved `get_random_variables` out of this function and into `simulators.py`